cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/scheduler_plugin_user/99-parallelcluster-scheduler-plugin.erb

```diff
@@ -1 +1,3 @@
 <%= node['cluster']['scheduler_plugin']['user'] %> ALL = (root) NOPASSWD: ALL
+
+<%= node['cluster']['cluster_admin_user'] %> ALL = (root) NOPASSWD: /usr/bin/cinc-client
```

Comment:
can we restrict this down so that pcluster-admin can run cinc-client as root only on a specific cookbook file?